libraries/chain/block_log.cpp

```diff
@@ -1557,22 +1547,21 @@ namespace eosio { namespace chain {
    }
 
    // static
-   void block_log::extract_blocklog(const fc::path& log_filename, const fc::path& index_filename,
-                                    const fc::path& dest_dir, uint32_t start_block_num, uint32_t num_blocks) {
+   void block_log::extract_block_range(const fc::path& block_dir, const fc::path& dest_dir,
+                                       block_num_type start_block_num, block_num_type end_block_num) {
```

Comment:
Generally, I'd prefer we use `end` as a synonym for 1 past `last` instead of `last`. This pattern is familiar to us and should be expected. Because we use end synonymous with last in other places in this file, I don't think it's necessary to change it in this one place unless we have bandwidth to change it throughout the file.